apps/game_backend/lib/game_backend/configuration.ex

```diff
@@ -541,4 +543,50 @@ defmodule GameBackend.Configuration do
   def get_latest_game_configuration do
     Repo.one(from(g in GameConfiguration, order_by: [desc: g.inserted_at], limit: 1))
   end
+
+  @doc """
+  Returns a 16 base encoded string representing the hashed value of the map configurations.
+  ## Examples
+      iex> get_configuration_hash_version()
+      "A6F0CC6917D195AB8A03129ACBE1FA48364845B8"
+  """
+  def get_configuration_hash_version do
+    get_current_version()
+    |> Map.get(:map_configurations)
+    |> Enum.flat_map(fn map_config ->
+      Map.take(map_config, [:obstacles, :initial_positions, :bushes, :radius]) |> Map.values()
+    end)
+    |> List.flatten()
+    |> Enum.map(fn config -> sum_shape_coordinates(config) end)
+    |> Enum.reduce(fn coordinates_last, coordinates_current -> Decimal.add(coordinates_last, coordinates_current) end)
+    |> Decimal.to_string()
+    |> (fn coordinates -> :crypto.hash(:sha, coordinates) |> Base.encode16() end).()
```

Comment:
Why base16?

Reply:
Just selected one at random. Any suggestions?

Reply:
Nope, was just curious. Also, don't think is worth discussing for this use case